core/src/main/java/tech/pegasys/web3signer/core/multikey/metadata/parser/SigningMetadataModule.java

```diff
@@ -81,4 +84,13 @@ public void serialize(
       gen.writeString(value.toString());
     }
   }
+
+  public static class NioPathSerializer extends JsonSerializer<Path> {
+    @Override
+    public void serialize(Path path, JsonGenerator gen, SerializerProvider serializers)
+        throws IOException {
+      // serialize Path without uri (Jackson's implementation uses uri)
+      gen.writeString(path.toString());
```

Comment:
Will this give us the absolute or relative path? I guess relative should be fine as long as it's relative to the keystore directory.